.github/workflows/node.js.yml

```
@@ -0,0 +1,61 @@
+name: Unit Tests
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
```

Only needed because you're using it below.

```suggestion
    runs-on: ubuntu-latest
    env:
      NPM_VERSION: "8.3.0"
```